tests/phpunit/tests/test-install.php

```diff
@@ -63,20 +67,22 @@ function test_uninstall() {
 
 		update_post_meta( $item_id, '_pll_menu_item', array() );
 
+		$this->assertFalse( defined( 'PLL_REMOVE_ALL_DATA' ), 'The constant PLL_REMOVE_ALL_DATA must not be defined for this test'  );
```

Comment:
I don't believe we should perform assertions on the test environment: we need to be absolutely sure about how we are running our test. Anyway, if the constant was defined, the test would fail at the next assertion...

Reply:
Same as above.